lbrynet/wallet/resolve.py

```diff
@@ -92,25 +88,16 @@ def __init__(self, claim_trie_root, height, transaction_class, hash160_to_addres
                 if 'height' in claim_response:
                     height = claim_response['height']
                     depth = self.height - height
-                    claim_result = _verify_proof(parsed_uri.name,
-                                                 claim_trie_root,
-                                                 claim_response,
-                                                 height, depth,
-                                                 transaction_class=self.transaction_class,
-                                                 hash160_to_address=self.hash160_to_address)
-                    result['claim'] = await self.parse_and_validate_claim_result(claim_result,
-                                                                                 certificate,
-                                                                                 raw)
-            elif claim_resolution_type == "claim_id":
-                result['claim'] = await self.parse_and_validate_claim_result(claim_response,
-                                                                             certificate,
-                                                                             raw)
-            elif claim_resolution_type == "sequence":
-                result['claim'] = await self.parse_and_validate_claim_result(claim_response,
-                                                                             certificate,
-                                                                             raw)
-            else:
-                log.error("unknown response type: %s", claim_resolution_type)
+                    claim_response = _verify_proof(parsed_uri.name,
+                                                   claim_trie_root,
+                                                   claim_response,
+                                                   height, depth,
+                                                   transaction_class=self.transaction_class,
+                                                   ledger=self.ledger)
```

Comment:
Same as the other comment about already having the `ledger` which has `ledger.transaction_class` (in fact, that's where you got it from originally) so you don't need to pass it separately.

Reply:
changing, good point